models/SmaAt_UNet.py

```diff
@@ -0,0 +1,58 @@
+from torch import nn
```

Comment:
Would you please add a brief documentation in the beginning of each of the file to describe what the code is doing?

Reply:
@junwang-noaa Added and changed `Purpose` to `Description`.